server/server.go

```diff
@@ -50,15 +50,21 @@ func Start(ip string, port int, cert string, key string, debug bool, t *Tracker,
 			continue
 		}
 
-		ip, _, _ := net.SplitHostPort(conn.RemoteAddr().String())
+		ip := getIP(conn)
 
 		context, err := limit.Get(nil, ip)
 		if err != nil {
+			if debugMode {
+				fmt.Printf("[Listener] Unable to get connection limit: %s\n", err)
+			}
 			conn.Close()
 			continue
 		}
 
 		if context.Reached {
+			if debugMode {
+				fmt.Printf("[Listener] %s broke the rate limit\n", ip)
```

Comment:
`fmt.Printf("[Listener] Rate limit exceeded by %st\n", ip)`

Reply:
active voice!

But I agree here. It's easier to read.